dace/codegen/targets/cpu.py

```diff
@@ -1456,7 +1472,8 @@ def _generate_NestedSDFG(
         ###############################################################
         # Write generated code in the proper places (nested SDFG writes
         # location info)
-        function_stream.write(global_code)
+        if not code_already_generated:
+            function_stream.write(global_code)
```

Comment:
not sure why you'd want to put that in a condition, since above you skip the generation of the function

Reply:
global_code is created in one of the ifs above